fiftyone/zoo/datasets/__init__.py

```diff
@@ -200,6 +309,12 @@ def load_zoo_dataset(
     :class:`fiftyone.core.dataset.Dataset` will be returned.
 
     Args:
+        name: the name of an existing zoo dataset or one to use for a
```

Comment:
I don't think we need both `name` and `gh_repo` for the case of loading a zoo dataset stored in a GH repo. My preference would be that the `name` argument should be extended to allow a github URL.

The existing `dataset_name` argument can be used if the user wants to control the name of the `Dataset` instance that is returned.

And I propose that, for github repos, `<user>/<name>` be used as the unique identifier for the zoo dataset, meaning that the dataset would be downloaded to `dataset_zoo_dir/<user>/<name>`.